ts/data/settings-key.ts

```diff
@@ -82,6 +90,13 @@ export const SettingsKey = {
   // NOTE: for these CTAs undefined means it has never been shown in this cycle of pro access, true means it needs to be shown and false means it has been shown and dont show it again.
   proExpiringSoonCTA: 'proExpiringSoonCTA',
   proExpiredCTA: 'proExpiredCTA',
+  // Proxy settings
+  proxyEnabled,
+  proxyType,
```

Comment:
I don't think `proxy-type` and `proxy-type` are used for anything currently, can be removed.